lib/Hakyll/Core/Compiler.hs

```diff
@@ -162,17 +162,17 @@ cached name compiler = do
     unless (resourceExists provider id') $ fail $ itDoesntEvenExist id'
 
     let modified = resourceModified provider id'
+        k = [name, show id']
+        go = compiler >>= \v -> v <$ compilerUnsafeIO (Store.set store k v)
```

Comment:
That's a nice one-liner, but I think a `do` block would have been much more readable (although slightly longer). Please restrain yourself a bit in the future :)

Reply:
You can definitely request me to change the style if you don't like it.  No hard feelings, really :)

Reply:
Augh... ghastly spelling error in the commit message too.  Whoops :(